### ISSUE TITLE
Feature request: add support for multilingual

### ISSUE THREAD

**Issue by ilovelinux** [tylersayshi/port-hugo#3]
Hi there! I am very satisfied with this theme, but now I need to add multilanguage support to my website.

Currently, the theme is using `data/content.yml` ([doc](https://gohugo.io/templates/data-templates/)) to fill the theme template. The issue is that data doesn't natively support internationalization.

I looked for solutions on the internet, and here's what I found:

- [This workaround](https://www.regisphilibert.com/blog/2018/08/hugo-multilingual-part-1-managing-content-translation/#data-files) would allows us to add support for internationalization while keep using `data`. It requires minor changes in the code and is not a breaking change. I already tried this approach and it works, I'm just having a minor issue with the logo but I'm already looking into it.
- We could switch to `contents` that [natively supports internazionalization](https://gohugo.io/content-management/multilingual/#translate-your-content). It looks like the best solution but requires a bit more effort. Also, I don't know why did you choose to use `data` and I want to be sure that the switch could be effective. This solution will introduce a breaking change.

We will also need to add something somewhere to allow users to change the website language. I was thinking about a drop-down list in the menu.

I'm available to create a PR to fix this issue once we choose the best solution for the project.

**Comment by tylersayshi** [comment]
Hey, I'd happily review and consider a PR! 

I avoided `contents` for the majority of the fields on the landing page because `contents` is setup to create multiple blog pages, so the concept doesn't really work for a single portfolio landing page.

The workaround seems fine to me. I might suggest adding a check for if `data/content.yaml` exists, use that, else check for `data/[intl]/content.yaml`...

**Comment by ilovelinux** [comment]
Glad to hear you!

In my opinion, we should first try to use data from `data/[intl]/content.yaml`, then use `data/content.yaml` config as fallback for undeclared configurations on the internationalized version.
Then, when there's no internationalization version, the page will be generated with `data/content.yaml`.

Using this approach, we also should be sure that every configuration set by `data/content.yaml` can be unset/reset to default by a `data/[intl]/.content.yaml` configuration entry.

Do you think it could work? Do you have any other ideas?

**Comment by tylersayshi** [comment]
Yeah that's fine with me. The order shouldn't really matter. Feel free to send a PR whenever
